strongbox-commons/src/test/java/org/carlspring/strongbox/config/PropertiesPathResolverConfigTest.java

```diff
@@ -0,0 +1,112 @@
+package org.carlspring.strongbox.config;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.core.env.Environment;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.carlspring.strongbox.config.PropertiesPathResolverConfig.PropertiesPathResolver;
+import static org.carlspring.strongbox.config.PropertiesPathResolverConfig.PropertiesPathResolver.PREFIX_OVERRIDE_PROPERTY;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PropertiesPathResolverConfigTest
+{
+    private static final String CURRENT_DIR = System.getProperty("user.dir");
+    private static final String CUSTOM_PATH_PROP = "strongbox.scary-t.file";
+    private static final String STRONGBOX_HOME_PROP = "strongbox.home";
+    private static final String STRONGBOX_HOME_DEFAULT = "/Users/keyboard-kitty/work/sbox";
+    private static final String DEFAULT_PATH = "etc/configzzz.yaml";
+
+    private Environment env;
+    private PropertiesPathResolver resolver;
+
+    @BeforeEach
+    public void setUp()
+    {
+        env = mock(Environment.class);
+        resolver = new PropertiesPathResolver(env);
+    }
+
+    @Test
+    public void testCustomAbsolutePath()
+    {
+        final String customPropValue = "/Users/sleepy/one/conf.yaml";
```

Comment:
As above.

Reply:
They are mock paths

Reply:
I see! Okay, thanks for the clarification! :)